push.sh

```diff
@@ -66,9 +66,16 @@ fi
 
 indent() { sed 's/^/  /'; }
 
+declare HELM3_VERSION="helm version | grep "v3\." | wc -l"
```

Comment:
Need to run in client mode only with `-c` or `--client`.  It may also be safer to run `--short` too.